04-onion-routing.md

```diff
@@ -575,7 +597,7 @@ HTLC at the final hop:
 2. data:
    * [`4`:`cltv_expiry`]
 
-If the `amt_to_forward` does not match the `incoming_htlc_amt` of
+If the `amt_to_forward` is higher than `incoming_htlc_amt` of
```

Comment:
But we're telling the next hop how much to forward so he'll just swallow that extra satoshi, mimicking the behavior of the last hop. The intermediate nodes will not fail with fee-too-high (otherwise the whole thing is way too brittle) so why should the last node?